src/posthog-core.ts

```diff
@@ -104,6 +106,8 @@ const defaultConfig = (): PostHogConfig => ({
     ui_host: null,
     token: '',
     autocapture: true,
+    // TODO: change to undefined when we release this so that remote config can turn it on even if unconfigured by the user
+    exception_autocapture: false,
```

Comment:
Feels a little weird that we have `exception_autocapture` locally and `autocaptureExceptions` remotely... Not sure if that means it should be different but it feels confusing...